### ISSUE TITLE
Keyboard should capitalize letters

### ISSUE THREAD

**Issue by andrewgaul** [kebernet/shortyz#84]
Presently the play board displays upper-case letters while the keyboard displays lower-case.

**Comment by kebernet** [comment]
Yeah you know. I have looked at this as a thing a couple of times. The thing is if I change one or other it looks weird to me. Capital letters on the keyboard seem really noisy. Lowercase letters in the puzzles seem hard to read and a bit un-natural (I never put lower case letters in a paper puzzle).

This might be something that could be a setting, though.
